CMakeLists.txt

```diff
@@ -240,9 +240,9 @@ set(INSTALL_LIB_DIR ${DEF_INSTALL_LIB_DIR} CACHE PATH "Installation directory fo
 
 # Never use RPATH. We have other means on every platform.
```

Comment:
Please remove the CMAKE lines associated with the Mac workarounds used previously if these are not needed anymore (which I am glad to see they are not).

Reply:
Almost forgotten about this, thanks for reminding.